charts/collabora-online/0.0.1/questions.yaml

```diff
@@ -0,0 +1,163 @@
+groups:
+  - name: "Container Image"
+    description: "Configure Container Image" 
+  - name: "Privileged"
+    description: "Privileged Configuration"
+  - name: "Networking"
+    description: "Network Configuration"
+  - name: "Environment Variables"
+    description: "Environment Variables Configuration"
+
+#Portal
+portals:
+  web_portal:
+    protocols:
+      - "https"
+    host: 
+      - "$node_ip"
+    ports:
+      - "$variable-service.port.nodePort"
+    path: "/loleaflet/dist/admin/admin.html"
```

Comment:
don't worry too much about the portal settings, i'll be working on standardising this part in the near future to deal with nodePort vs Ingress